feeds/rubygems/rubygems_test.go

```diff
@@ -119,4 +115,7 @@ func rubyGemsPackagesResponse(w http.ResponseWriter, r *http.Request) {
 ]
 
 `))
+	if err != nil {
+		fmt.Println("Unexpected error during mock http server write: %w", err)
```

Comment:
In a test this should raise the HTTP error similarly to the implementation of HttpServe outside of the tests (duplicate)